src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostInitiateByIdControllerTest.java

```diff
@@ -496,11 +496,11 @@ void should_return_201_with_task_assigned() throws Exception {
                 jsonPath("$.task_id").value(taskId),
```

Comment:
can we rename test as state is now UNASSIGNED?

Reply:
done